src/btree/bt_split.c

```diff
@@ -1508,19 +1465,7 @@ __wt_split_multi(WT_SESSION_IMPL *session, WT_REF *ref, int exclusive)
 		 mod->write_gen = 0;
 		 __wt_cache_dirty_decr(session, page);
 	}
-	__wt_ref_out(session, ref);
-
-	/*
-	 * We may not be able to immediately free the page's original WT_REF
-	 * structure and instantiated key, there may be threads using them.
-	 * Add them to the session discard list, to be freed once we know it's
-	 * safe.
-	 */
-	if (ikey != NULL)
-		WT_TRET(__split_safe_free(session, split_gen, exclusive,
-		    ikey, sizeof(WT_IKEY) + ikey->size));
-	WT_TRET(__split_safe_free(session, split_gen, exclusive,
-	    ref, sizeof(WT_REF)));
+	__wt_page_out(session, &page);
```

Comment:
@michaelcahill, why switch from `__wt_ref_out` to `__wt_page_out`? I don't think it's a problem, but I don't understand the need, either.